Code/Samples/ShaderExplorer/ShaderExplorer.cpp

```diff
@@ -230,6 +234,7 @@ void ezShaderExplorerApp::AfterCoreSystemsStartup()
   ezGlobalLog::AddLogWriter(ezLogWriter::Console::LogMessageHandler);
   ezGlobalLog::AddLogWriter(ezLogWriter::VisualStudio::LogMessageHandler);
 
+  ezTelemetry::CreateServer();
```

Comment:
This should already be done by the application base class somewhere.

Reply:
See comment on the other ezTelemetry function.